plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -307,7 +307,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 
 				rbacv1helpers.NewRule(Write...).Groups(policyGroup).Resources("poddisruptionbudgets").RuleOrDie(),
 
-				rbacv1helpers.NewRule(Write...).Groups(networkingGroup).Resources("networkpolicies", "ingresses").RuleOrDie(),
+				rbacv1helpers.NewRule(Write...).Groups(networkingGroup).Resources("networkpolicies", "ingresses", "clustercidrconfigs").RuleOrDie(),
```

Comment:
Why does aggregate-to-edit need to write CCC?  The comment says " a role for a namespace level editor.  It grants access to all user level actions in a namespace."

Reply:
Removed, not required. Leftovers from local debugging, cleaned up now